tests/testing/test_testing.py

```diff
@@ -255,13 +270,17 @@ def get_session_data(request: Request) -> Dict[str, Any]:
 
     app = Starlite(route_handlers=[get_session_data], middleware=[session_config.middleware])
 
-    with TestClient(app=app, session_config=session_config) as client:
+    with TestClient(app=app, session_config=session_config, backend=test_client_backend) as client:
         client.set_session_data(session_data)
         assert session_data == client.get("/test").json()
 
 
 @pytest.mark.parametrize("with_domain", [False, True])
-def test_test_client_get_session_data(with_domain: bool, session_config: "BaseBackendConfig") -> None:
+def test_test_client_get_session_data(
+    with_domain: bool,
```

Comment:
Was this test using this argument earlier?

Reply:
As a matter of fact it wasn't. It should have, but it didn't. I just tried adding it but it fails because of the issue described in #706. Will add an `xfail` for now.

Reply:
Addressed